libs/application-generic/src/usecases/get-subscriber-global-preference-v1/get-subscriber-global-preference-v1.usecase.ts

```diff
@@ -0,0 +1,101 @@
+import { Injectable } from '@nestjs/common';
+import {
+  PreferenceLevelEnum,
+  SubscriberEntity,
+  SubscriberPreferenceRepository,
+  SubscriberRepository,
+} from '@novu/dal';
+
+import { IPreferenceChannels, ChannelTypeEnum } from '@novu/shared';
+import { GetSubscriberGlobalPreferenceCommandV1 } from './get-subscriber-global-preference-v1.command';
+import { buildSubscriberKey, CachedEntity } from '../../services/cache';
+import { ApiException } from '../../utils/exceptions';
+import { InstrumentUsecase } from '../../instrumentation';
+
+/** @deprecated - use `GetPreferences` instead */
```

Comment:
Please elaborate on this until we migrate. We should GetPreferences apart from the places that the deprecated usecase is already used, right?

Reply:
I added some more clarity [here](https://github.com/novuhq/novu/pull/6817/files#diff-b85899d0434586dd3aeeb1910c51c1742571347dc8ff8f396ea9396670dfdca4R15-R23).

Let me know if that clears up the approach.